### ISSUE TITLE
Alphabetical reference for all tag names

### ISSUE THREAD

**Issue by rentonxp** [google/earthenterprise#1064]
Hello,

Is there an alphabetical reference for all tag names used by fusion in xml files ?

Thx

**Comment by tst-basilhuffman** [comment]
Sure, please consult the fusion administration section of the documentation. Probably the most relevant portion: http://www.opengee.org/geedocs/5.2.4/answer/176738.html

**Comment by tst-basilhuffman** [comment]
@rentonxp Was this the information you were looking for? Feel free to join the opengee slack channel  @ opengee.slack.com also

**Comment by rentonxp** [comment]
Not exactly, i'm trying to understand how tag names are working in khasset.xml files. 
Looking for something similar to this :
[KML Reference guide](https://developers.google.com/kml/documentation/kmlreference)

**Comment by tst-ppenev** [comment]
I don't think those files have DTDs.  They're basically XML serializations of C++ objects.  I think your best bet is looking at the code.  The code for all the numerous types of assets is not very easy to look through.  A lot of the final classes are generated by Perl scripts during a build.
